baselines/fedht/fedht/aggregate.py

```diff
@@ -0,0 +1,137 @@
+# Copyright 2020 Flower Labs GmbH. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Aggregation functions for strategy implementations."""
+# mypy: disallow_untyped_calls=False
+
+from functools import reduce
+from typing import List, Tuple
+
+import numpy as np
+from flwr.common import NDArrays
+
+def weighted_loss_avg(results: List[Tuple[int, float]]) -> float:
+    """Aggregate evaluation results obtained from multiple clients."""
+    num_total_evaluation_examples = sum(num_examples for (num_examples, _) in results)
+    weighted_losses = [num_examples * loss for num_examples, loss in results]
+    return sum(weighted_losses) / num_total_evaluation_examples
```

Comment:
I believe this function is not used. let's remove?

Reply:
removed